ui/apps/platform/src/Containers/SystemHealth/Components/DeclarativeConfigurationHeathCard.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect, ReactElement } from 'react';
+import {
+    Alert,
+    Card,
+    CardBody,
+    CardHeader,
+    CardHeaderMain,
+    CardTitle,
+    Flex,
+    FlexItem,
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Thead, Th, Tr } from '@patternfly/react-table';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import pluralize from 'pluralize';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+import { fetchDeclarativeConfigurationsHealth } from 'services/IntegrationHealthService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+import { IntegrationHealthItem } from '../utils/integrations';
+
+type Props = {
+    pollingCount: number;
+};
+
+function DeclarativeConfigurationHealthCard({ pollingCount }: Props): ReactElement {
+    const [isFetching, setIsFetching] = useState(false);
+    const [requestErrorMessage, setRequestErrorMessage] = useState('');
+    const [items, setItems] = useState<IntegrationHealthItem[]>([]);
+
+    useEffect(() => {
+        setIsFetching(true);
+        fetchDeclarativeConfigurationsHealth()
+            .then((itemsFetched) => {
+                setRequestErrorMessage('');
+                setItems(itemsFetched);
+            })
+            .catch((error) => {
+                setRequestErrorMessage(getAxiosErrorMessage(error));
+                setItems([]);
+            })
+            .finally(() => {
+                setIsFetching(false);
+            });
+    }, [pollingCount]);
+
+    /*
+     * Wait for isFetching only until response to the initial request.
+     * Otherwise count temporarily disappears during each subsequent request.
+     */
+    const hasCount = (pollingCount !== 0 || !isFetching) && !requestErrorMessage;
+    const itemCount = items.length;
+
+    return (
+        <Card isFullHeight isCompact>
+            <CardHeader>
+                <CardHeaderMain>
+                    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                        <FlexItem>
+                            <CardTitle component="h2">Declarative configuration</CardTitle>
```

Comment:
```suggestion
                            <CardTitle component="h2">Declarative Configuration</CardTitle>
```

Reply:
> (suggestion) Declarative Configuration

The PatternFly style guide calls for heading to use "Sentence case", where only the first word is capitalized.